wsd/CacheUtil.hpp

```diff
@@ -23,6 +23,13 @@ struct CacheQuery
     std::string _uri;
     std::string _stamp;
     std::string _dest;
+
+    CacheQuery(const std::string& uri, const std::string stamp, const std::string dest)
```

Comment:
would it be better to be all "const std::string&" here?

Reply:
Ah yes, let me fix. I just happened to have clang-15 here and after pulling, I ran into this where old enough compilers need an explicit ctor for some reason.